RSDKv5/RSDK/Core/Reader.cpp

```diff
@@ -263,7 +266,7 @@ bool32 RSDK::LoadFile(FileInfo *info, const char *filename, uint8 fileMode)
         }
         if (modSettings.activeMod != -1) {
             PrintLog(PRINT_NORMAL, "[MOD] Failed to find file %s in active mod %s", filename, modList[m].id.c_str());
-            // TODO return false? check original impl later
+            break; // no one ever checked the original implementation... :pensive:
```

Comment:
This skips all following mods in the mod list.

Reply:
i see, weirdly enough that is how the other decomps do the stuff related to that, but yeah i'll try to look into it further